python/cugraph/pagerank/test_pagerank.py

```diff
@@ -116,18 +140,22 @@ def networkx_call(M, max_iter, tol, alpha):
 MAX_ITERATIONS = [500]
 TOLERANCE = [1.0e-06]
 ALPHA = [0.85]
+PERSONALIZATION_PERC = [0, 10, 50]
```

Comment:
I think we should add a comment explaining how these parameters impact the way the personalization is generated in `networkx_call `.

Reply:
Fixed [here](https://github.com/rapidsai/cugraph/pull/300/files?file-filters%5B%5D=.h&file-filters%5B%5D=.md&file-filters%5B%5D=.py&file-filters%5B%5D=.sh&owned-by%5B%5D=#diff-83a484342317d80b4509f185eee9a5bbR85)